distributed/http/scheduler/api.py

```diff
@@ -24,6 +24,16 @@ async def post(self):
             else:
                 workers = params.get("workers", {})
                 workers_info = await scheduler.retire_workers(workers=workers)
+
+            # digests_total_since_heartbeat contains tuples as keys which are not json serializable
+            def clean_dict(d):
```

Comment:
I wonder if `distributed.utils.recursive_to_dict()` would be suitable instead of defining this here.